### DISCUSSION TITLE
Logging all SQL queries

### DISCUSSION THREAD

**Discussion by trengrj** [emmett-framework/emmett#414] [General]
Is there an easy way to log all SQL queries in emmett including how long each query took? I like to see SQL queries while developing an application to ensure they are optimized.

**Comment by gi0baro** [comment]
@trengrj initialising the `Database` class with the `debug` param should logs everything executed.

```python
db = Database(app, debug=True)
```

**Reply by trengrj** [reply]
Thanks, exactly what I needed! I took a look at the `Database` class but didn't realise there was a flag via inheritance from `DAL`.
